src/sidebar/test/group-fixtures.js

```diff
@@ -0,0 +1,172 @@
+'use strict';
+
+const Chance = require('chance');
+const chance = new Chance();
+
+function group () {
```

Comment:
Small stylistic nit. We avoid a space between the function name and '('. Anonymous functions in existing code are written as `function ()` (new code should generally prefer arrow functions).

Reply:
Note that the space between function name and paren is something that can be added to an `eslintrc` or whatnot—the reason there _is_ a space is that I started a spike on this stuff outside of the repo and it was using my own `eslintrc` setup—which mandates the space :).

Reply:
> Anonymous functions in existing code are written as function () (new code should generally prefer arrow functions).

Is this just informational? Or did you spot somewhere where I'm not using arrow functions where I should be?

Reply:
> Is this just informational? 

Just informational.

> Note that the space between function name and paren is something that can be added to an eslintrc or whatnot

Yes. Incidentally I would be quite happy for us to adopt [Prettier](https://github.com/prettier/prettier) and never need to discuss such matters in code review again.